2018-09-19 14:18:32.672557: E tensorflow/stream_executor/cuda/cuda_driver.cc:397] failed call to cuInit: CUDA_ERROR_UNKNOWN
2018-09-19 14:18:32.672613: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:150] kernel driver does not appear to be running on this host (Studentroompc01): /proc/driver/nvidia/version does not exist
2018-09-19 14:18:37.471501: E tensorflow/core/common_runtime/executor.cc:697] Executor failed to create kernel. Invalid argument: Conv2DCustomBackpropFilterOp only supports NHWC.
	 [[Node: gradients_2/Discriminator.3_1/Conv2D_grad/Conv2DBackpropFilter = Conv2DBackpropFilter[T=DT_FLOAT, _class=["loc:@gradients_2/AddN_12"], data_format="NCHW", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 2, 2], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](Maximum_4, ConstantFolding/gradients_2/Discriminator.3_1/Conv2D_grad/ShapeN-matshapes-1, gradients_2/AddN_1, ^gradients_2/Discriminator.3_1/BiasAdd_grad/BiasAddGrad)]]
Using TensorFlow backend.
Uppercase local vars:
	BATCH_SIZE: 64
	CONTINUE: False
	CRITIC_ITERS: 5
	CURRENT_PATH: sintel/cgantfssim
	DIM: 64
	IM_DIM: 32
	ITERS: 100000
	LAMBDA: 10
	MODE: wgan-gp
	OUTPUT_DIM: 3072
	SQUARE_IM_DIM: 1024
	START_ITER: 0
load lists from files
load lists from files
Creating train generator with 1018 samples.
Traceback (most recent call last):
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1278, in _do_call
    return fn(*args)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1263, in _run_fn
    options, feed_dict, fetch_list, target_list, run_metadata)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1350, in _call_tf_sessionrun
    run_metadata)
tensorflow.python.framework.errors_impl.InvalidArgumentError: Conv2DCustomBackpropFilterOp only supports NHWC.
	 [[Node: gradients_2/Discriminator.3_1/Conv2D_grad/Conv2DBackpropFilter = Conv2DBackpropFilter[T=DT_FLOAT, _class=["loc:@gradients_2/AddN_12"], data_format="NCHW", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 2, 2], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](Maximum_4, ConstantFolding/gradients_2/Discriminator.3_1/Conv2D_grad/ShapeN-matshapes-1, gradients_2/AddN_1, ^gradients_2/Discriminator.3_1/BiasAdd_grad/BiasAddGrad)]]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "../../../cgan_sintel_ssim.py", line 239, in <module>
    _disc_cost, _ = session.run([disc_cost, disc_train_op], feed_dict={real_data_int: _real_data, cond_data_int: _cond_data})
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 877, in run
    run_metadata_ptr)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1100, in _run
    feed_dict_tensor, options, run_metadata)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1272, in _do_run
    run_metadata)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1291, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.InvalidArgumentError: Conv2DCustomBackpropFilterOp only supports NHWC.
	 [[Node: gradients_2/Discriminator.3_1/Conv2D_grad/Conv2DBackpropFilter = Conv2DBackpropFilter[T=DT_FLOAT, _class=["loc:@gradients_2/AddN_12"], data_format="NCHW", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 2, 2], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](Maximum_4, ConstantFolding/gradients_2/Discriminator.3_1/Conv2D_grad/ShapeN-matshapes-1, gradients_2/AddN_1, ^gradients_2/Discriminator.3_1/BiasAdd_grad/BiasAddGrad)]]

Caused by op 'gradients_2/Discriminator.3_1/Conv2D_grad/Conv2DBackpropFilter', defined at:
  File "../../../cgan_sintel_ssim.py", line 159, in <module>
    disc_train_op = tf.train.AdamOptimizer(learning_rate=1e-4, beta1=0.5, beta2=0.9).minimize(disc_cost, var_list=disc_params)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/training/optimizer.py", line 400, in minimize
    grad_loss=grad_loss)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/training/optimizer.py", line 514, in compute_gradients
    colocate_gradients_with_ops=colocate_gradients_with_ops)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/gradients_impl.py", line 596, in gradients
    gate_gradients, aggregation_method, stop_gradients)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/gradients_impl.py", line 779, in _GradientsHelper
    lambda: grad_fn(op, *out_grads))
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/gradients_impl.py", line 398, in _MaybeCompile
    return grad_fn()  # Exit early
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/gradients_impl.py", line 779, in <lambda>
    lambda: grad_fn(op, *out_grads))
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/nn_grad.py", line 529, in _Conv2DGrad
    data_format=data_format)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/gen_nn_ops.py", line 1091, in conv2d_backprop_filter
    dilations=dilations, name=name)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/util/deprecation.py", line 454, in new_func
    return func(*args, **kwargs)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 3155, in create_op
    op_def=op_def)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 1717, in __init__
    self._traceback = tf_stack.extract_stack()

...which was originally created as op 'Discriminator.3_1/Conv2D', defined at:
  File "../../../cgan_sintel_ssim.py", line 117, in <module>
    disc_fake = Discriminator(fake_data, cond_data)
  File "../../../cgan_sintel_ssim.py", line 94, in Discriminator
    output = lib.ops.conv2d.Conv2D('Discriminator.3', 2*DIM, 4*DIM, 5, output, stride=2)
  File "/home/linkermann/Desktop/MA/opticalFlow/opticalFlowGAN/tflib/ops/conv2d.py", line 111, in Conv2D
    data_format='NCHW'
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/gen_nn_ops.py", line 956, in conv2d
    data_format=data_format, dilations=dilations, name=name)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/util/deprecation.py", line 454, in new_func
    return func(*args, **kwargs)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 3155, in create_op
    op_def=op_def)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 1717, in __init__
    self._traceback = tf_stack.extract_stack()

InvalidArgumentError (see above for traceback): Conv2DCustomBackpropFilterOp only supports NHWC.
	 [[Node: gradients_2/Discriminator.3_1/Conv2D_grad/Conv2DBackpropFilter = Conv2DBackpropFilter[T=DT_FLOAT, _class=["loc:@gradients_2/AddN_12"], data_format="NCHW", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 2, 2], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](Maximum_4, ConstantFolding/gradients_2/Discriminator.3_1/Conv2D_grad/ShapeN-matshapes-1, gradients_2/AddN_1, ^gradients_2/Discriminator.3_1/BiasAdd_grad/BiasAddGrad)]]

2018-09-19 14:22:51.189363: E tensorflow/stream_executor/cuda/cuda_driver.cc:397] failed call to cuInit: CUDA_ERROR_UNKNOWN
2018-09-19 14:22:51.189408: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:150] kernel driver does not appear to be running on this host (Studentroompc01): /proc/driver/nvidia/version does not exist
2018-09-19 14:22:55.331730: E tensorflow/core/common_runtime/executor.cc:697] Executor failed to create kernel. Invalid argument: Conv2DCustomBackpropFilterOp only supports NHWC.
	 [[Node: gradients_2/Discriminator.3_1/Conv2D_grad/Conv2DBackpropFilter = Conv2DBackpropFilter[T=DT_FLOAT, _class=["loc:@gradients_2/AddN_12"], data_format="NCHW", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 2, 2], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](Maximum_4, ConstantFolding/gradients_2/Discriminator.3_1/Conv2D_grad/ShapeN-matshapes-1, gradients_2/AddN_1, ^gradients_2/Discriminator.3_1/BiasAdd_grad/BiasAddGrad)]]
Using TensorFlow backend.
Uppercase local vars:
	BATCH_SIZE: 64
	CONTINUE: False
	CRITIC_ITERS: 5
	CURRENT_PATH: sintel/cgantfssim
	DIM: 64
	IM_DIM: 32
	ITERS: 100000
	LAMBDA: 10
	MODE: wgan-gp
	OUTPUT_DIM: 3072
	SQUARE_IM_DIM: 1024
	START_ITER: 0
load lists from files
load lists from files
Creating train generator with 1018 samples.
Traceback (most recent call last):
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1278, in _do_call
    return fn(*args)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1263, in _run_fn
    options, feed_dict, fetch_list, target_list, run_metadata)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1350, in _call_tf_sessionrun
    run_metadata)
tensorflow.python.framework.errors_impl.InvalidArgumentError: Conv2DCustomBackpropFilterOp only supports NHWC.
	 [[Node: gradients_2/Discriminator.3_1/Conv2D_grad/Conv2DBackpropFilter = Conv2DBackpropFilter[T=DT_FLOAT, _class=["loc:@gradients_2/AddN_12"], data_format="NCHW", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 2, 2], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](Maximum_4, ConstantFolding/gradients_2/Discriminator.3_1/Conv2D_grad/ShapeN-matshapes-1, gradients_2/AddN_1, ^gradients_2/Discriminator.3_1/BiasAdd_grad/BiasAddGrad)]]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "../../../cgan_sintel_ssim.py", line 239, in <module>
    _disc_cost, _ = session.run([disc_cost, disc_train_op], feed_dict={real_data_int: _real_data, cond_data_int: _cond_data})
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 877, in run
    run_metadata_ptr)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1100, in _run
    feed_dict_tensor, options, run_metadata)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1272, in _do_run
    run_metadata)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/client/session.py", line 1291, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.InvalidArgumentError: Conv2DCustomBackpropFilterOp only supports NHWC.
	 [[Node: gradients_2/Discriminator.3_1/Conv2D_grad/Conv2DBackpropFilter = Conv2DBackpropFilter[T=DT_FLOAT, _class=["loc:@gradients_2/AddN_12"], data_format="NCHW", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 2, 2], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](Maximum_4, ConstantFolding/gradients_2/Discriminator.3_1/Conv2D_grad/ShapeN-matshapes-1, gradients_2/AddN_1, ^gradients_2/Discriminator.3_1/BiasAdd_grad/BiasAddGrad)]]

Caused by op 'gradients_2/Discriminator.3_1/Conv2D_grad/Conv2DBackpropFilter', defined at:
  File "../../../cgan_sintel_ssim.py", line 159, in <module>
    disc_train_op = tf.train.AdamOptimizer(learning_rate=1e-4, beta1=0.5, beta2=0.9).minimize(disc_cost, var_list=disc_params)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/training/optimizer.py", line 400, in minimize
    grad_loss=grad_loss)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/training/optimizer.py", line 514, in compute_gradients
    colocate_gradients_with_ops=colocate_gradients_with_ops)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/gradients_impl.py", line 596, in gradients
    gate_gradients, aggregation_method, stop_gradients)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/gradients_impl.py", line 779, in _GradientsHelper
    lambda: grad_fn(op, *out_grads))
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/gradients_impl.py", line 398, in _MaybeCompile
    return grad_fn()  # Exit early
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/gradients_impl.py", line 779, in <lambda>
    lambda: grad_fn(op, *out_grads))
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/nn_grad.py", line 529, in _Conv2DGrad
    data_format=data_format)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/gen_nn_ops.py", line 1091, in conv2d_backprop_filter
    dilations=dilations, name=name)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/util/deprecation.py", line 454, in new_func
    return func(*args, **kwargs)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 3155, in create_op
    op_def=op_def)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 1717, in __init__
    self._traceback = tf_stack.extract_stack()

...which was originally created as op 'Discriminator.3_1/Conv2D', defined at:
  File "../../../cgan_sintel_ssim.py", line 117, in <module>
    disc_fake = Discriminator(fake_data, cond_data)
  File "../../../cgan_sintel_ssim.py", line 94, in Discriminator
    output = lib.ops.conv2d.Conv2D('Discriminator.3', 2*DIM, 4*DIM, 5, output, stride=2)
  File "/home/linkermann/Desktop/MA/opticalFlow/opticalFlowGAN/tflib/ops/conv2d.py", line 111, in Conv2D
    data_format='NCHW'
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/ops/gen_nn_ops.py", line 956, in conv2d
    data_format=data_format, dilations=dilations, name=name)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/util/deprecation.py", line 454, in new_func
    return func(*args, **kwargs)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 3155, in create_op
    op_def=op_def)
  File "/home/linkermann/anaconda2/envs/MA/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 1717, in __init__
    self._traceback = tf_stack.extract_stack()

InvalidArgumentError (see above for traceback): Conv2DCustomBackpropFilterOp only supports NHWC.
	 [[Node: gradients_2/Discriminator.3_1/Conv2D_grad/Conv2DBackpropFilter = Conv2DBackpropFilter[T=DT_FLOAT, _class=["loc:@gradients_2/AddN_12"], data_format="NCHW", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 1, 2, 2], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](Maximum_4, ConstantFolding/gradients_2/Discriminator.3_1/Conv2D_grad/ShapeN-matshapes-1, gradients_2/AddN_1, ^gradients_2/Discriminator.3_1/BiasAdd_grad/BiasAddGrad)]]

